webapp/client/src/features/datasetVersions/store/actions.ts

```diff
@@ -114,7 +114,7 @@ export const loadDatasetVersion = (
   dispatch(action(loadDatasetVersionActionTypes.REQUEST, { id }));
 
   await ServiceFactory.getDatasetVersionsService()
-    .loadDatasetVersion(id, datasetId)
+    .loadDatasetVersion(workspaceName, id, datasetId)
```

Comment:
@CaptEmulation : this fixes a issue i pinged on #webapp on slack , not sure if it need a better treatment, if so feel free to commit to the PR.

Reply:
THANK YOU. I was looking at the exact line and scratching my head very recently

Reply:
well actually I was looking at another usage of this method where it was not including the `datasetId`.

In any case, this LGTM :+1: